source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -93,6 +93,68 @@ The operations are categorized by purpose and functionality.
    from the Query API expression provided in each example. However,
    both expressions will produce the same aggregation result.
 
+Arithmetic Operations
+~~~~~~~~~~~~~~~~~~~~~
+
+You can use the methods described in this section to perform arithmetic
+operations on values of type ``MqlInteger`` or ``MqlNumber``.
```

Comment:
Maybe an introductory sentence that doesn't reference types would be useful here, something like:

"You can use arithmetic operations to manipulate numbers. You can perform an arithmetic operation on a value of type ``MqlInteger`` or ``MqlNumber`` using the methods described in this section."